### ISSUE TITLE
Dataset download

### ISSUE THREAD

**Issue by seba-eng** [CVUsers/Smoke-Detect-by-YoloV5#17]
Hi, thanks for the interesting project, I think I managed to understand most of your findings using google translate. Are you able to provide a download link for the data on a free platform other than baidu? I can't access it from Europe. Thanks for your help!

**Comment by CVUsers** [comment]
I can't use Google.
As mentioned in an issue, I put the data on foreign accessible websites:
https://download.csdn.net/download/qq_46098574/75444687

The platform rules can not be free,soory

---Original---
From: "Sebastian ***@***.***&gt;
Date: Tue, May 10, 2022 17:41 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [CVUsers/Smoke-Detect-by-YoloV5] Dataset download (Issue #17)




 
Hi, thanks for the interesting project, I think I managed to understand most of your findings using google translate. Are you able to provide a download link for the data on a free platform other than baidu? I can't access it from Europe. Thanks for your help!

**Comment by seba-eng** [comment]
Thanks for the fast response! Geo-blocking really sucks :( 
Could you maybe try to put the dataset directly here into the github repo? As far as I know the hard size limit from Github is quite high, you can easily upload a few gigabyte.

**Comment by CVUsers** [comment]
Okay, I'll go back after work and upload it on my computer.



---Original---
From: "Sebastian ***@***.***&gt;
Date: Tue, May 10, 2022 17:58 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVUsers/Smoke-Detect-by-YoloV5] Dataset download (Issue #17)




 
Thanks for the fast response! Geo-blocking really sucks :(
 Could you maybe try to put the dataset directly here into the github repo? As far as I know the hard size limit from Github is quite high, you can easily upload a few gigabyte.

**Comment by RiverChu0** [comment]
Don't expect too much. The purpose of this project is to collect fees, and it is not a real open source sharing project. You are in Europe, you are not his target user, and he will not really care about your request.

**Comment by CVUsers** [comment]
You're wrong. I've already given it to him for free. I've given it to more than 3,000 people for free. It's just that he's in Europe, and I can't use google driver here, so it's on csdn. There's a link on it, and I'll give you a quote from Lu Xun: I never guess others with malice.



---Original---
From: "River ***@***.***&gt;
Date: Fri, Sep 9, 2022 04:22 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVUsers/Smoke-Detect-by-YoloV5] Dataset download (Issue #17)




 
Don't expect too much. The purpose of this project is to collect fees, and it is not a real open source sharing project. You are in Europe, you are not his target user, and he will not really care about your request.
